custom_components/mail_and_packages/const.py

```diff
@@ -733,6 +756,43 @@
         icon="mdi:package-variant-closed",
         key="dhl_parcel_nl_packages",
     ),
+    # Post NL
+    "post_nl_processing": SensorEntityDescription(
+        name="Post NL Processing",
+        native_unit_of_measurement="package(s)",
+        icon="mdi:dolly",
+        key="post_nl_preparing",
+    ),
+    "post_nl_collection": SensorEntityDescription(
+        name="Post NL Ready for Collection",
+        native_unit_of_measurement="package(s)",
+        icon="mdi:store-clock",
+        key="post_nl_ready_for_collection",
+    ),
```

Comment:
These are states we don't track.

Reply:
I think this would be nice to add however, since parcel shop collection can happen for failed parcels

Reply:
It's mostly out of the scope of the integration, we're only really interested in "heading to your home", "delivered to your home", "exceptions delivering your package" (if applicable), and the count of packages.

Reply:
Still think it would be a nice addition, the integration comes across as parcel tracking, and this is a vital step in the process

Reply:
I'd start by remove the changes in `helpers.py` any added on sensors should fall out of the bottom of the `if` tree.

If you want to make sure the processing works correctly you'll need to supply a raw email, sanitized of any personal information, into the tests with the reset of the test emails.

Then you can use another test as an example of how to test your changes.
Example:

https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/dev/tests/test_helpers.py#L1069-L1074

and

This is where you'd read in the email for the above test:

https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/dev/tests/conftest.py#L452-L476